### ISSUE TITLE
Delay Publication for COGA Review

### ISSUE THREAD

**Issue by brewerj** [w3c/wai-inclusion#25]
This is a comment related to the page [Accessibility, Usability, and Inclusion](//w3c.github.io/wai-inclusion/index.html) Intersection between accessibility, usability, and COGA input

I think that this document should be reviewed by COGA before being published, especially the definition of usability with relation to understandability, which intersects with some cognitive accessibility issues.

**Comment by shawna-slh** [comment]
Shawn sent e-mail to WAI Coordination Call list on 9 Dec, and to Lisa Seeman on 13 Dec with subject "[timely] COGA, accessibility, usability review".

**Comment by yatil** [comment]
As we haven’t heard back in 6 weeks, I think we should publish the document and revisit it later in case comments come up.

**Comment by sharronrush** [comment]
+1 to Eric with a courtesy note to COGA saying just that

**Comment by nitedog** [comment]
I'd like to close this too.

**Comment by yatil** [comment]
It would be good if we could clear this up asap for including the page into the redesign.

**Comment by sharronrush** [comment]
Hey @yatil - I think we made the decision to publish. I will take on the task of communicating to Lisa the fact that hearing no objection we went forward and that we are completely open to any edits she wants to suggest. So, please publish and close this issue. Thank you.
